launch.sh

```diff
@@ -14,6 +16,3 @@ su postgres -c 'psql -f /apollo/user.sql'
 
 su postgres -c 'PGPASSWORD=apollo psql -U apollo -h 127.0.0.1 chado -f /chado.sql'
 
```

Comment:
If postgres is being removed, this file can mostly be removed as well. Just need to re-instate the copy statement.